Shared/Samples/Run valve isolation trace/RunValveIsolationTraceView.swift

```diff
@@ -0,0 +1,152 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct RunValveIsolationTraceView: View {
+    /// The view model for the sample.
+    @StateObject var model = Model()
+    
+    /// The last locations in the screen and map where a tap occurred.
+    @State var lastSingleTap: (screenPoint: CGPoint, mapPoint: Point)?
+    
+    /// A Boolean value indicating if the configuration sheet is presented.
+    @State var isConfigurationPresented = false
+    
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(
+                map: model.map,
+                graphicsOverlays: [model.parametersOverlay]
+            )
+            .onSingleTapGesture { screenPoint, mapPoint in
+                lastSingleTap = (screenPoint, mapPoint)
+            }
+            .overlay(alignment: .top) {
+                Text(model.statusText)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.ultraThinMaterial, ignoresSafeAreaEdges: .horizontal)
+                    .multilineTextAlignment(.center)
+            }
+            .task {
+                do {
+                    try await model.setup()
+                } catch {
+                    model.statusText = error.localizedDescription
+                }
```

Comment:
The setup and trace functions are both async throws.

However, we call and treat those inconsistently. Here the setup function is nested in a do/catch and the error is caught and the status is updated on the model.

The pattern with the trace() function is the error is ignored but the trace function internally updates the status.

I prefer the pattern with the trace function because the model handles everything, but then trace does not need to be marked as throws.

So model.setup() also should not be throws.  If it encounters an error it should update the status internally.